libs/execution_context/ebpf_maps.c

```diff
@@ -1175,6 +1193,7 @@ _initialize_lru_entry(
     ebpf_assert(_get_key_state(map, partition, entry) == EBPF_LRU_KEY_UNINITIALIZED);
```

Comment:
Is this assert needed (since we assert uninitialized for all partitions below)?

Reply:
Nope, it's redundant. Will remove